### PR TITLE
Indicate units for disk size.

### PR DESCRIPTION
**Pull request by frewsxcv**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

File: _reference/rack-parameters.md [hunk 1]
```diff
@@ -86,6 +86,6 @@ Have the Rack API use an Internal ELB, making it unreachable from the Internet.
 
 ## VolumeSize
 
-Default disk size of the EBS volume attached to each EC2 instance in the cluster.
+Default disk size (in gibibytes) of the EBS volume attached to each EC2 instance in the cluster.
```
Comment (mwarkentin) [inline _reference/rack-parameters.md]: That's a new one to me. 😄 

`gigabytes`, or just `GB`?

Comment (frewsxcv) [inline _reference/rack-parameters.md]: I'm not sure I understand your comment. The AWS docs say GiB, not GB

Comment (frewsxcv) [inline _reference/rack-parameters.md]: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-blockdev-template.html

Comment (mwarkentin) [inline _reference/rack-parameters.md]: Oh wow, you're right.. I'd never heard of gibibytes before. My bad!

**Pull request merged**